style/rules.css

```diff
@@ -0,0 +1,14 @@
+.rules {
+  margin-top: 10rem;
+  padding: 3rem 5rem;
+  border: 2px solid var(--color-primary);
+  border-radius: 1rem;
+}
+
+.rules-info {
+  display: flex;
+  align-items: center;
+}
+p > span {
+  margin-right: 1rem;
+}
```

Comment:
use class for targetting rather than an element.

Reply:
will make changes